phase4-profile-bdew/src/main/java/com/helger/phase4/profile/bdew/BDEWCompatibilityValidator.java

```diff
@@ -509,14 +511,36 @@ public void validateUserMessage (@Nonnull final Ebms3UserMessage aUserMsg, @Nonn
       if (aFrom != null)
       {
         if (aFrom.getPartyIdCount () > 1)
+        {
           aErrorList.add (_createError ("PartyInfo/From must contain no more than one PartyID"));
+        }
+
+        if (StringHelper.hasNoText (aFrom.getRole ()) || !aFrom.getRole ().equals (CAS4.DEFAULT_INITIATOR_URL))
```

Comment:
That's the same as `if (!CAS4.DEFAULT_INITIATOR_URL.equals (aFrom.getRole ()))` right?

Reply:
Yes, that's true. Thank you for the proposal (see new commit)